pkg/scheduler/plugins/gang/gang.go

```diff
@@ -120,7 +120,11 @@ func (gp *gangPlugin) OnSessionOpen(ssn *framework.Session) {
 		}
 
 		if !lReady && !rReady {
-			if lv.UID < rv.UID {
+			if lv.PodGroup.GetCreationTimestamp().Time.Equal(rv.PodGroup.GetCreationTimestamp().Time) {
```

Comment:
hm... we need to add a new field into JobInfo, the creation-timestamp is from PodGroup or PDB. And we will check jobInfo's creation time here. We need to support PDB as tf-operator is using it, we will remove it at least after 0.4 release.

Reply:
Something along these lines ? (https://github.com/kubernetes-sigs/kube-batch/pull/464/commits/3f0dfd856741f6d0365962fdd6b652eb06ede2e3)